admin/sql/updates/20160516-mbs-8720.sql

```diff
@@ -0,0 +1,35 @@
+\set ON_ERROR_STOP 1
+BEGIN;
+
+CREATE TABLE edit_note_recipient (
+    editor_id    INTEGER NOT NULL, -- PK, references editor.id
+    edit_note_id INTEGER NOT NULL  -- PK, references edit_note.id
+);
+
+ALTER TABLE edit_note_recipient ADD CONSTRAINT edit_note_recipient_pkey PRIMARY KEY (editor_id, edit_note_id);
+
+ALTER TABLE edit_note_recipient
+   ADD CONSTRAINT edit_note_recipient_fk_editor
+   FOREIGN KEY (editor_id)
+   REFERENCES editor(id);
+
+ALTER TABLE edit_note_recipient
+   ADD CONSTRAINT edit_note_recipient_fk_edit_note
+   FOREIGN KEY (edit_note_id)
+   REFERENCES edit_note(id);
+
+CREATE INDEX edit_note_recipient_idx_editor_id ON edit_note_recipient (editor_id);
+
+-- Trigger
```

Comment:
We'll need the function + trigger to go in a separate file, say `20160516-mbs-8720-triggers.sql`, which only runs on standalone servers. Don't want these to be modifying `edit_note_recipient` on slave servers since that table will be replicated. (Then in upgrade.json just add the -triggers file to the standalone list.)